### PR TITLE
Add tests for Meeting

### PR DESCRIPTION
**Pull request by ThomasHoooo**

### CODE REVIEW

**Review by junweimoo — APPROVED** [review]
Looks good!

File: src/main/java/seedu/address/model/ClientBook.java [hunk 1]
```diff
@@ -93,6 +97,14 @@ public void removeClient(Client key) {
         clients.remove(key);
     }
 
+    /**
+     * Adds a client to the client book.
+     * The client must not already exist in the client book.
+     */
```
Comment (sikai00) [inline src/main/java/seedu/address/model/ClientBook.java:103]: This should be add meeting in the docs

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -99,21 +108,21 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
      * Executes the given {@code command}, confirms that <br>
      * - a {@code CommandException} is thrown <br>
      * - the CommandException message matches {@code expectedMessage} <br>
-     * - the client book, filtered client list and selected client in {@code actualModel} remain unchanged
+     * - the address book, filtered person list and selected person in {@code actualModel} remain unchanged
```
Comment (sikai00) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:111]: Not sure why is it back to address book when the object is ClientBook already

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -99,21 +108,21 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
...
 
         assertThrows(CommandException.class, expectedMessage, () -> command.execute(actualModel));
-        assertEquals(expectedClientBook, actualModel.getClientBook());
+        assertEquals(expectedAddressBook, actualModel.getClientBook());
         assertEquals(expectedFilteredList, actualModel.getFilteredClientList());
     }
     /**
-     * Updates {@code model}'s filtered list to show only the client at the given {@code targetIndex} in the
-     * {@code model}'s client book.
+     * Updates {@code model}'s filtered list to show only the person at the given {@code targetIndex} in the
```
Comment (sikai00) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:124]: same issue

**Review by sikai00 — COMMENTED** [review]
just some simple semantics issues

**Pull request merged**